src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/CourtService.kt

```diff
@@ -226,7 +226,7 @@ class CourtService(
     court: String?
   ): List<VideoLinkBookingResponse> {
     val scheduledAppointments = prisonApiService
-      .getScheduledAppointmentsByAgencyAndDate(agencyId, date)
+      .getScheduledAppointmentsByAgencyAndDate(agencyId, date, null, null)
```

Comment:
I think we need a new method name for the extra parameters.

Reply:
i did consider that (it would have meant less a lot less typing), but I didn't see any overloaded methods anywhere else so i bit the bullet.